input/vocabulary/valueset-signal-filerepo-classification.json

```diff
@@ -0,0 +1,524 @@
+{
+    "resourceType": "ValueSet",
+    "id": "signal-filerepo-classification",
+    "url": "https://signalbhn.org/fhir/us/core/ValueSet/signal-filerepo-classification",
+    "name": "SignalFileRepoClassification",
+    "title": "Signal FileRepo Classification",
+    "status": "draft",
+    "experimental": false,
+    "date": "2023-09-26",
+    "description": "Classification of File Repository",
+    "compose": {
+        "include": [
+            {
+                "system": "https://signalbhn.org/fhir/CodeSystem/signal-filerepo-classification",
+                "concept": [
```

Comment:
You could have made a conditional to include all  codes in this code system, but listing them out should work.